InfoMovies/InfoMovies/Views/ItemDetailsPage.xaml.cs

```diff
@@ -0,0 +1,34 @@
+﻿using InfoMovies.ViewModels;
+using System;
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace InfoMovies.Views
+{
+    [XamlCompilation(XamlCompilationOptions.Compile)]
+	public partial class ItemDetailsPage : BasePage
+	{
+		public ItemDetailsPage (object viewModel)
+		{
+            BindingContext = viewModel;
```

Comment:
How can we improve this?

Reply:
now, i dont can think a way different that:

`new ItemDetailPage(){ BindingContext = viewModel};`

I searched about another ways to do this, but dont find a "better way" for this
any suggest?  😆